rules/line-between-sibling-blocks.js

```diff
@@ -0,0 +1,109 @@
+            const includesData = includes ? organizeData(content, includes) : null;
+            const closingTagsData = closingTags ? organizeData(content, closingTags) : null;
+
+            return {
+                blockTagsData: blockTagsData,
+                includesData: includesData,
+                closingTagsData: closingTagsData,
+            }
+        };
+
+        const findEntryIndex = (startIndex, content, entry) => {
+            const index = content.slice(startIndex).indexOf(entry);
+
+            return startIndex + index;
+        };
+
+        const organizeData = (content, tags) => tags.map((tag, index, arr) => {
+                const entryIndex = findEntryIndex(currentIndex, content, tag);
+                currentIndex = entryIndex + tag.length;
+
+                if (index === arr.length - 1) {
+                    currentIndex = 0;
+                }
+
+                return {
+                    data: tag,
+                    index: entryIndex,
+                }
+            });
+
+        this.filterModulesData(data, 'twig').forEach(twigFileData => {
+            const { files, type, name } = twigFileData;
+            const { twig } = files;
+            const { path, api } = twig;
+            const { external } = api;
+            const blockNames = (external && external.blocks) ? external.blocks : null;
+            const getValidationString = (closingTagData, nextBlockData) =>
+                twig.content.slice(closingTagData.index + closingTagData.data.length, nextBlockData.index);
+
+            if (!blockNames) {
+                return;
+            }
+
+            const structureData = contentParser(twig.content);
+            const { blockTagsData, includesData, closingTagsData } = structureData;
+            const getClosestElement = previousBlock => item => item.index > previousBlock.index;
+            const checkSymbolsBetweenBlocks = (endOfTopBlock, nextBlock) => {
+                if (!nextBlock) {
+                    return;
+                }
+
+                const stringBetweenBlocks = getValidationString(endOfTopBlock, nextBlock);
+                const countLineBreaks = 3;
+                const symbolRegExp = /[a-zA-Z0-9\'\"\>]/;
+                const stringsArrayLength = stringBetweenBlocks.split('\n').length;
+
+                if (symbolRegExp.test(stringBetweenBlocks)) {
+                    return;
+                }
+
+                if (stringsArrayLength < countLineBreaks) {
+                    addError(formatMessage(`Please add empty line before ${nextBlock.data}`, type, name, path));
+                }
+
+                if (stringsArrayLength > countLineBreaks) {
+                    addError(formatMessage(`Please remove unnecessary empty lines before ${nextBlock.data}`, type, name, path));
+                }
+            };
+
+            if (closingTagsData) {
+                closingTagsData.forEach(closingTagData => {
```

Comment:
Please rename `closingTagsData` for example add `list` word, because difficult to read `closingTagsData` and `closingTagData`.

Reply:
I renamed closingTagsData to closingTagsList